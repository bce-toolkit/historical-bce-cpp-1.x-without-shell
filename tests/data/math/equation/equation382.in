7 6
1 0 0 0 -1 0 0
4 4 0 0 -46 -1 0
-3 -2 1 1 0 0 0
0 1 0 0 -12 0 0
0 0 1 0 -3 0 0
0 0 4 1 -24 -2 0
# PO4(3e-)+MoO4(2e-)+NH4(e+)+H(e+)=(NH4)3(P(Mo12O40)).6H2O+H2O
