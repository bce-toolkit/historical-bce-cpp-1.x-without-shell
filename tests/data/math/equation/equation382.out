1/6Xa
2Xa
1/2Xa
4Xa
1/6Xa
Xa
